problemsets-mat223/modules/module11.tex

```diff
@@ -376,10 +376,14 @@
 trying to be extra precise, we must make sure our boxes/lists of numbers and our transformations/vectors stay separate.
 
 \begin{example}
-	Let $\mathcal T$ be the transformation induced by the matrix $M=\mat{1&2&5\\2&-2&-2}$,
+	Let $\mathcal T$ be the transformation induced by the matrix $M=[\mathcal T_{M}\vec v]_{\mathcal E'}= M[\vec v]_{\mathcal E}$,
 	and let $\vec v=3\xhat-3\zhat$. Compute $\mathcal T(\vec v)$.
 
-	XXX Finish
+	Since $\mathcal T$ is induced by $M=\mat{1&2&5\\2&-2&-2}$, by definition, 	
+	\[
+	    [\mathcal T_{M}\vec v]_{\mathcal E'}= M[\vec v]_{\mathcal E}=\mat{1&2&5\\2&-2&-2}\mat{3\\0\\-3}=\matc{-12\\12}.
```

Comment:
This example needs to be explained more. Write `[\mathcal T_{M}\vec v]_{\mathcal E'}= M[\vec v]_{\mathcal E}` and then explain, `[\vec v]_{\mathcal E}=\mat{3\\0\\-3}` and so ...